### DISCUSSION TITLE
2.4

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#26] [公告]
## 错误修复

* Transmission 模块得到了全面大修
  * 解决了 TR 的下载任务一旦暂停就无法恢复的问题
  * 解决了 QB 的封禁任务可能串到 TR 并导致报错的问题
  * TR 现在也支持 verifySSL 配置文件参数来忽略自签 SSL 证书错误

### 介绍全新 WebUI 界面

新的 WebUI 界面允许显示您所连接的各个客户端的最后一次 API 请求状态，并能够显示目前 PBH 为您托管的封禁列表。  
WebUI 能够展示以下内容：
* 下载器名称
* 下载器 API 端点
* 下载器最新请求状态
* 下载器活动种子数量
* 下载器活动对等体数量
* 循环检查次数
* 封禁对等体数量
* 解封对等体数量
* 对等体列表
  * 对等体 IP 地址
  * 对等体 User-Agent
  * 对等体 PeerId
  * 对等体封禁时间
  * 对等体预计解封时间
  * 对等体于哪个种子上被检测到并被封禁
  * 对等体在封禁时的上传/下载总量（Transmission不可用），以及下载进度
  * 封禁描述

![demo](https://github.com/Ghost-chu/PeerBanHelper/assets/30802565/f59f3ef2-98bc-441e-8a38-50388f51d84c)

<hr /><em>This discussion was created from the release <a href='https://github.com/Ghost-chu/PeerBanHelper/releases/tag/v2.4'>2.4</a>.</em>

**Comment by Mishasama** [comment]
这次怎么又把 jar 打包到一起了…… 我差点以为 exe 又变成启动器了😂
